lib/rubocop/cop/rspec/let_before_examples.rb

```diff
@@ -43,14 +43,27 @@ class LetBeforeExamples < Base
           }
         PATTERN
 
+        # @!method include_examples?(node)
+        def_node_matcher :include_examples?, <<~PATTERN
+          {
+            #{block_pattern(':include_examples')}
+            #{send_pattern(':include_examples')}
+          }
+        PATTERN
+
         def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
           return unless example_group_with_body?(node)
+          return if example_group_with_included_examples?(node.body)
 
           check_let_declarations(node.body) if multiline_block?(node.body)
         end
 
         private
 
+        def example_group_with_included_examples?(body)
+          body.children.find { |sibling| include_examples?(sibling) }
```

Comment:
since we are not interested in the actual include_examples node, I'd replace `find` for `any?`